stories/FormField-stories.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import type { Meta, StoryObj } from '@storybook/react';
+import type { ComponentProps } from 'react';
+import { FormField, FormFieldValidation, FormFieldWidth } from '../src/components/form/FormField';
+
+type FormField = typeof FormField;
+
+const meta: Meta<FormField> = {
+  component: FormField,
+  title: 'Echoes/Forms/FormField (internal)',
+  argTypes: {
+    validation: {
+      control: { type: 'select' },
+      options: Object.values(FormFieldValidation),
+      table: {
+        defaultValue: { summary: FormFieldValidation.None },
+      },
+    },
+    width: {
+      control: { type: 'select' },
+      options: Object.values(FormFieldWidth),
+      table: {
+        defaultValue: { summary: FormFieldWidth.Full },
+      },
+    },
+  },
+  parameters: {
+    docs: {
+      toc: true,
+    },
+  },
+};
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
I don't see this issue? Maybe you are using the wrong eslint config? All our stories export it like this.